VaticanBAV/et/BAVet46.xml

```diff
@@ -42,14 +42,18 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            <ptr target="bm:Lusini1996Ewostatewos"/>
                            <citedRange unit="page">354-356</citedRange>
                         </bibl>
+                        <bibl>
+                          <ptr target="bm:Taddesse1972Church"/>
+                          <citedRange unit="page">208f</citedRange>
+                        </bibl>
```

Comment:
Could you please provide the whole range of pages and add a change element with the information on adding a bibliographical refernce.